cdap-security-spi/src/main/java/co/cask/cdap/security/spi/authorization/PrivilegesManager.java

```diff
@@ -33,28 +34,63 @@
    * @param entity the {@link EntityId} to whom {@link Action actions} are to be granted
    * @param principal the {@link Principal} that performs the actions. This could be a user, or role
    * @param actions the set of {@link Action actions} to grant.
+   * @deprecated As of release 4.3.0, replaced by {@link #grant(Authorizable, Principal, Set)}
    */
+  @Deprecated
   void grant(EntityId entity, Principal principal, Set<Action> actions) throws Exception;
 
   /**
+   * Grants a {@link Principal} authorization to perform a set of {@link Action actions} on an {@link EntityId}.
+   * Note: this grant is used to support wildcard privilege management
+   *
+   * @param authorizable the {@link Authorizable} to whom {@link Action actions} are to be granted
```

Comment:
`The {@link Authorizable} on which the {@link Action actions} are to be granted.